=== STDOUT ===
Set parameter Username
Academic license - for non-commercial use only - expires 2026-09-01
Set parameter TimeLimit to value 300
Gurobi Optimizer version 11.0.0 build v11.0.0rc2 (linux64 - "Ubuntu 22.04.5 LTS")

CPU model: Intel(R) Core(TM) Ultra 7 155H, instruction set [SSE2|AVX|AVX2]
Thread count: 11 physical cores, 22 logical processors, using up to 22 threads

Optimize a model with 384226 rows, 37824 columns and 1135036 nonzeros
Model fingerprint: 0xe73499a6
Variable types: 17874 continuous, 19950 integer (19950 binary)
Coefficient statistics:
  Matrix range     [1e+00, 5e+02]
  Objective range  [1e+00, 1e+00]
  Bounds range     [1e+00, 1e+00]
  RHS range        [1e+00, 1e+00]
Presolve removed 88939 rows and 6435 columns (presolve time = 9s) ...
Presolve removed 88939 rows and 6435 columns (presolve time = 10s) ...
Presolve removed 88939 rows and 6435 columns (presolve time = 19s) ...
Presolve removed 88939 rows and 6435 columns
Presolve time: 19.57s
Presolved: 295287 rows, 31389 columns, 855911 nonzeros
Variable types: 17037 continuous, 14352 integer (14352 binary)
Found heuristic solution: objective 14928.000000
Found heuristic solution: objective 14920.000000

Deterministic concurrent LP optimizer: primal simplex, dual simplex, and barrier
Showing barrier log only...

Root relaxation presolved: 31389 rows, 326676 columns, 887300 nonzeros

Root barrier log...

Ordering time: 0.03s

Barrier statistics:
 Free vars  : 1893
 AA' NZ     : 4.827e+05
 Factor NZ  : 3.129e+06 (roughly 170 MB of memory)
 Factor Ops : 3.150e+08 (less than 1 second per iteration)
 Threads    : 9

                  Objective                Residual
Iter       Primal          Dual         Primal    Dual     Compl     Time
   0  -1.85712623e+06 -0.00000000e+00  1.19e+04 1.00e+00  2.23e+01    22s
   1  -1.14127111e+06  4.42062227e+03  2.40e+03 1.67e+00  1.21e+01    22s
   2  -7.70440673e+05  8.62668228e+03  3.12e-11 7.53e-01  4.93e+00    22s
   3  -4.92716052e+05  7.82383994e+03  1.05e-11 5.88e-02  1.65e+00    22s
   4  -6.21598394e+04  7.00178416e+03  4.97e-12 5.28e-05  2.02e-01    22s

Barrier performed 4 iterations in 22.28 seconds (8.36 work units)
Barrier solve interrupted - model solved by another algorithm

Concurrent spin time: 0.12s

Solved with dual simplex

Root relaxation: objective 0.000000e+00, 3324 iterations, 1.83 seconds (1.34 work units)

    Nodes    |    Current Node    |     Objective Bounds      |     Work
 Expl Unexpl |  Obj  Depth IntInf | Incumbent    BestBd   Gap | It/Node Time

     0     0    0.00000    0   54 14920.0000    0.00000   100%     -   29s
H    0     0                    10568.000000    0.00000   100%     -   30s
H    0     0                    10311.000000    0.00000   100%     -   31s
     0     0    0.00000    0   65 10311.0000    0.00000   100%     -   39s
H    0     0                    7486.0000000    0.00000   100%     -   40s
H    0     0                    6384.0000000    0.00000   100%     -   41s
     0     0    0.00000    0   63 6384.00000    0.00000   100%     -   58s
H    0     0                    6366.0000000    0.00000   100%     -   60s
     0     0    0.00000    0   64 6366.00000    0.00000   100%     -   61s
     0     0    0.00000    0   78 6366.00000    0.00000   100%     -  100s
H    0     0                    5602.0000000    0.00000   100%     -  101s
     0     0    0.00000    0   74 5602.00000    0.00000   100%     -  103s
     0     0    0.00000    0   71 5602.00000    0.00000   100%     -  122s
H    0     0                    5590.0000000    0.00000   100%     -  124s
     0     0    0.00000    0   71 5590.00000    0.00000   100%     -  125s
     0     0    0.00000    0   66 5590.00000    0.00000   100%     -  146s
     0     0    0.00000    0   66 5590.00000    0.00000   100%     -  156s
H    0     0                    5120.0000000    0.00000   100%     -  185s
H    0     0                    4212.0000000    0.00000   100%     -  185s
H    0     2                    3750.0000000    0.00000   100%     -  198s
     0     2    0.00000    0   66 3750.00000    0.00000   100%     -  198s
     1     4    0.00000    1   67 3750.00000    0.00000   100%  5344  224s
     3     7    0.00000    2   54 3750.00000    0.00000   100% 18564  265s
     7    11  194.00000    3 1120 3750.00000    0.00000   100% 13155  300s

Cutting planes:
  Gomory: 1
  Cover: 6
  Implied bound: 417
  Clique: 2951
  MIR: 68
  StrongCG: 1
  RLT: 308
  Relax-and-lift: 922

Explored 12 nodes (174301 simplex iterations) in 300.12 seconds (563.94 work units)
Thread count was 22 (of 22 available processors)

Solution count 10: 3750 4212 5120 ... 10568

Time limit reached
Best objective 3.750000000000e+03, best bound 0.000000000000e+00, gap 100.0000%
Backend tkagg is interactive backend. Turning interactive mode on.


=== STDERR ===
Traceback (most recent call last):
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/attach_pid_injected.py", line 75, in attach
    debugpy.listen(setup["address"])
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/public_api.py", line 47, in wrapper
    return wrapped(*args, **kwargs)
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 133, in debug
    log.reraise_exception("{0}() failed:", func.__name__, level="info")
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 131, in debug
    return func(address, settrace_kwargs, **kwargs)
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 145, in listen
    raise RuntimeError("debugpy.listen() has already been called on this process")
RuntimeError: debugpy.listen() has already been called on this process
E+00253.767: Traceback (most recent call last):
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/attach_pid_injected.py", line 75, in attach
                 debugpy.listen(setup["address"])
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/public_api.py", line 47, in wrapper
                 return wrapped(*args, **kwargs)
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 133, in debug
                 log.reraise_exception("{0}() failed:", func.__name__, level="info")
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 131, in debug
                 return func(address, settrace_kwargs, **kwargs)
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 145, in listen
                 raise RuntimeError("debugpy.listen() has already been called on this process")
             RuntimeError: debugpy.listen() has already been called on this process
             
             Stack where logged:
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/bin/carta", line 8, in <module>
                 sys.exit(main_cli())
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/carta/carta.py", line 176, in main_cli
                 main(arguments)
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/_vendored/pydevd/pydevd.py", line 2188, in do_wait_suspend
                 keep_suspended = self._do_wait_suspend(thread, frame, event, arg, trace_suspend_type, from_this_thread, frames_tracker)
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/_vendored/pydevd/pydevd.py", line 2257, in _do_wait_suspend
                 notify_event.wait(wait_timeout)
               File "/home/nischayvagarwal/.pyenv/versions/3.10.13/lib/python3.10/threading.py", line 607, in wait
                 signaled = self._cond.wait(timeout)
               File "/home/nischayvagarwal/.pyenv/versions/3.10.13/lib/python3.10/threading.py", line 324, in wait
                 gotit = waiter.acquire(True, timeout)
               File "<string>", line 1, in <module>
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/attach_pid_injected.py", line 90, in attach
                 log.reraise_exception()
               File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/common/log.py", line 222, in reraise_exception
                 _exception(format_string, *args, **kwargs)
             

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/attach_pid_injected.py", line 90, in attach
    log.reraise_exception()
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/attach_pid_injected.py", line 75, in attach
    debugpy.listen(setup["address"])
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/public_api.py", line 47, in wrapper
    return wrapped(*args, **kwargs)
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 133, in debug
    log.reraise_exception("{0}() failed:", func.__name__, level="info")
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 131, in debug
    return func(address, settrace_kwargs, **kwargs)
  File "/home/nischayvagarwal/ugp/github/UGP-Diff-Map-Inference/CARTA/venv/lib/python3.10/site-packages/debugpy/server/api.py", line 145, in listen
    raise RuntimeError("debugpy.listen() has already been called on this process")
RuntimeError: debugpy.listen() has already been called on this process
